docs/DeveloperGuide.md

```diff
@@ -926,6 +926,75 @@ testers are expected to do more *exploratory* testing.
         - These three actions violates EstateEase's constraints against duplicate houses, person and incorrect format, hence making the `addressbook.json` corrupted.
         - The application should automatically detect this, and display an empty EstateEase.
 
+[//]: # (@@author redcolorbicycle)
+### Adding a house to a seller
+
+**Prerequisites:** 
+- There is a seller by the name of "John Doe".
+- There is no seller by the name of "Lim Carl".
+- A Condominium with Street "Clementi Ave 2", no Block, Level "02", Unit Number 25, Postal Code 578578 and Price of 99999 does not currently exist in EstateEase.
+- A Condominium with Street "Clementi Ave 2", no Block, Level "05", Unit Number 25, Postal Code 578578 and Price of 99999 currently exists in EstateEase.
+
+1. **Adding the house to the seller**
+
+    1. **Test case:** ` addHouse n/John Doe type/Condominium street/Clementi Ave 2 blk/N/A level/02 unitNo/25 postal/578578 price/99999`
+       **Expected:** New house added!
+
+1. **Invalid house format**
+
+    1. **Test case:** ` addHouse n/John Doe type/Condominium Ave 2 blk/N/A level/02 unitNo/25 postal/578578 price/99999`
+       **Expected:** Invalid command format!
+
+    1. **Test case:** ` addHouse n/John Doe type/Condominium street/Clementi Ave 2 blk/N/A unitNo/25 postal/578578 price/99999`
+       **Expected:** Invalid command format!
+   
+    1. **Test case:** ` addHouse n/John Doe type/Condominium street/Clementi Ave 2 blk/N/A level/02 postal/578578 price/99999`
+       **Expected:** Invalid command format!
+
+    1. **Test case:** ` addHouse n/John Doe type/Condominium street/Clementi Ave 2 blk/N/A level/02 unitNo/25 price/99999`
+       **Expected:** Invalid command format!
+
+1. **Invalid seller**
+
+    1. **Test case:** `addHouse n/Lim Carl type/Condominium street/Clementi Ave 2 blk/N/A level/02 unitNo/25 postal/578578 price/99999`
+       **Expected:** This Seller does not exist in EstateEase
+
+1. **House already exists**
+
+    1. **Test case:** `addHouse n/John Doe type/Condominium street/Clementi Ave 2 blk/N/A level/05 unitNo/25 postal/578578 price/99999`
+       **Expected:** This house already exists in EstateEase
```

Comment:
Missing ```<br>``` tag after each `Test case` input.

It would probably look like this:
![image](https://github.com/AY2324S2-CS2103-F09-1/tp/assets/42115432/0f38644e-f8cb-4c8d-bddd-075047794aaa)

On that note, I think we need to fix the format for matching buyer manual testing too 🤣

Reply:
Yes thanks for pointing it out!